src/OpenTelemetry/README.md

```diff
@@ -84,6 +84,51 @@ using var otel = Sdk.CreateTracerProvider(b => b
   * [Building your own Processor](../../docs/trace/extending-the-sdk/README.md#processor)
   * [Building your own Sampler](../../docs/trace/extending-the-sdk/README.md#sampler)
 
+### Troubleshooting
+
+Self diagnostics module is provided to help troubleshooting.
+
+To enable self diagnostics, go to the current directory of your process and
+create a configuration file named `OTEL_DIAGNOSTICS.json` with the following
+content:
+
+```json
+{
+    "LogDirectory": ".",
+    "FileSize": 1024,
+    "EventLevel": "Error"
+}
+```
+
+`LogDirectory` is the directory of the output file. It can be an absolute path
+or a relative path to the current directory. `FileSize` is a positive integer,
+which specifies the log file size in
+[KiB](https://en.wikipedia.org/wiki/Kibibyte).
+`EventLevel` is the lowest level of the events to be captured.
```

Comment:
I guess the terms might be confusing - e.g. logs vs events (e.g. why LogDirectory instead of EventDirectory, why EventLevel instead of LogLevel). I understand that there is a history, would like to get more feedback before we go with the names.

Reply:
https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/sdk-environment-variables.md#general-sdk-configuration
Calling it LogLevel would be more aligned with what spec calls the ENV variable.

Reply:
I've renamed it as `LogLevel`. The paragraph explained that the value has to be one of the values of the [`EventLevel` enum](https://docs.microsoft.com/dotnet/api/system.diagnostics.tracing.eventlevel), but in this context, `LogLevel` as the configuration name makes more sense.